src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -257,6 +295,32 @@ class HaAutomationEditor extends LitElement {
     );
   }
 
+  private _toggleYaml(): void {
+    this._showYaml = !this._showYaml;
+  }
+
+  private get _yamlEditor(): HuiYamlEditor | null {
+    return this.shadowRoot!.querySelector("hui-yaml-editor");
+  }
+
+  private get _yaml(): string {
```

Comment:
Don't make this a getter, on every render (every time a property changes) it will dump the `json` config to `yaml`. Instead just make it a property.